### ISSUE TITLE
[Request]:  Let us be able to give cars too our friends 

### ISSUE THREAD

**Issue by ZeroGTAV** [Yimura/YimMenu#3049]
### Problem

Im not sure if tthis is a thing but if so please ell me how to do it

### Solution

a script hat yim can use or another way

### Reason

it would be nice to help our friends and other players out who are new 

### Additional context

N/A

**Comment by Ahmad23145** [comment]
Download the script - https://github.com/Deadlineem/Extras-Addon-for-YimMenu

**Comment by heni015** [comment]
Yea it's working. Confirmed.
To make sure I got full garage (2slots for vehicles) and friend also got, but full with shitty cars taken from street.
Select him from players list, spawn choosen car with no selected "NO POLLUTION".
Get in to his 'new' car, customize it with LSCustom TAB in Yim and get out, when friend get in just press "Gift Car" if success he can go to garage and switch car from his fully garage. :d
